styles.scss

```diff
@@ -68,10 +92,24 @@ a {
   padding-bottom: 20px;
 }
 
+.preview-link {
+  padding-top: 0;
+  padding-bottom: 0;
+}
+
 .hero-banner {
-  background-image: url('./assets/about-page_hero-banner.svg');
-  background-size: auto 100%;
+  background-image: url('./assets/bg.png');
+  background-size: cover;
   background-position: center;
+  color: white;
+  height: 480px;
+
```

Comment:
On wide displays, the headers aren't each on a newline. "Philly's 2nd" and Social Justice Hackathon are put side-by-side.

<img width="1701" alt="image" src="https://github.com/socialjusticehackathon/website/assets/2574498/3e623d44-5076-4101-90a2-ed612cc6ffa0">

The suggested CSS below ensures it wraps on wide displays

```suggestion
  display: flex;
  /* Uncomment below to vertically center text */
  /*justify-content: center;*/
  align-items: center;
  flex-direction: column;
  /* outline letters in black */
  /* from https://stackoverflow.com/a/4772914 */
  text-shadow: 1px 0px 0 #000, 0 -1px 0 #000, 0 1px 0 #000, -1px 0 0 #000;

```

Here is the result:

<img width="1702" alt="image" src="https://github.com/socialjusticehackathon/website/assets/2574498/a4db92ab-5720-4c17-b227-2d78ff822a44">

Reply:
Note that text-shadow is a little bit funky for anything other than 1 px outline. There's the `-website-text-stroke` property that's pretty widely supported, but I haven't tried that.